server/routes/admin/mailboxes.js

```diff
@@ -49,7 +49,7 @@ module.exports = ({ configClient }) => (router, audited) => {
       return res.redirect(`/admin/mailboxes/edit/${id}`)
     }
 
-    res.redirect('/admin/mailboxes')
+    return res.redirect('/admin/mailboxes')
```

Comment:
typescript warns about inconsistent returns if some places use an explicit return and other places don't

Reply:
but I didn't think that `res.redirect` returns anything, so `return`ing the value from it looks weird

Reply:
We inherited this style from form app starter in use-of-force.. I think it's a common convention to return `res.redirect` in node apps, e.g:

<img width="1302" alt="Screenshot 2020-04-17 at 14 42 34" src="https://user-images.githubusercontent.com/1517745/79575426-b14add00-80b9-11ea-82e3-48d60c709cf9.png">